src/script/common/c_content.cpp

```diff
@@ -526,6 +526,12 @@ ContentFeatures read_content_features(lua_State *L, int index)
 	// Amount of light the node emits
 	f.light_source = getintfield_default(L, index,
 			"light_source", f.light_source);
+	if (f.light_source > LIGHT_MAX) {
+		warningstream << "Node " << f.name.c_str()
+			<< " had greater light_source than 14, it was reduced."
+			<< std::endl;
+		f.light_source = LIGHT_MAX;
+	}
```

Comment:
If a modder made a mod that overwrote minetest.register_node(), that modder could crash the server with a too high light value. It causes a segmentation fault which is hard to debug, so IMO it is better to check twice than have a bad error.